docs/source/userguide/command_line.rst

```diff
@@ -1,508 +1,520 @@
 .. dropdown:: Distribution Statement
 
- | # # # This source code is protected under the license referenced at
- | # # # https://github.com/NRLMMD-GEOIPS.
+ This source code is protected under the license referenced at
+ https://github.com/NRLMMD-GEOIPS.
 
 .. _command_line:
 
-Command Line Interface
-**********************
+Command Line Interface (CLI)
+****************************
 
-.. _geoips:
+.. warning::
 
-:ref:`GeoIPS CLI <geoips>`
+    The CLI is currently under development.
+    The CLI may change without warning!
+    Please consult this documentation for up-to-date info on the CLI.
 
-The GeoIPS Command Line Interface (CLI) is currently under development, but we'll keep
-updated instructions in this documentation that reflect the current state of the CLI.
+The CLI interacts with GeoIPS artifacts.
+A GeoIPS artifact is a piece of data or functionality,
+such as an Interface, Package, Plugin,
+Family, Test Script, Test Dataset, etc.
 
```

Comment:
A different thought here:

> Interaction with and introspection of GeoIPS is performed using the CLI. The CLI can be used to run, configure, and test GeoIPS. It can also print lists of available artifacts and descriptions of individual artifacts.

Reply:
This is a bit clunky: "Interaction with and introspection of GeoIPS is performed using the CLI." Can we remove it and just leave it at "The CLI can be used to run, configure, and test GeoIPS. It can also print lists or descriptions of available data or functionality."